app/upgrades/v15/upgrades.go

```diff
@@ -18,6 +20,113 @@ import (
 	poolmanagertypes "github.com/osmosis-labs/osmosis/v14/x/poolmanager/types"
 )
 
+// an array of 10 strings containing the numbers from 1 to 10:
+
+// Using the same queries as stargate_whitelist.go
+var whitelistedQueries = []string{
+	// cosmos-sdk queries
+
+	// auth
+	"/cosmos.auth.v1beta1.Query/Account",
+	"/cosmos.auth.v1beta1.Query/Params",
+
+	// bank
+	"/cosmos.bank.v1beta1.Query/Balance",
+	"/cosmos.bank.v1beta1.Query/DenomMetadata",
+	"/cosmos.bank.v1beta1.Query/Params",
+	"/cosmos.bank.v1beta1.Query/SupplyOf",
+
+	// distribution
+	"/cosmos.distribution.v1beta1.Query/Params",
+	"/cosmos.distribution.v1beta1.Query/DelegatorWithdrawAddress",
+	"/cosmos.distribution.v1beta1.Query/ValidatorCommission",
+
+	// gov
+	"/cosmos.gov.v1beta1.Query/Deposit",
+	"/cosmos.gov.v1beta1.Query/Params",
+	"/cosmos.gov.v1beta1.Query/Vote",
+
+	//slashing
+	"/cosmos.slashing.v1beta1.Query/Params",
+	"/cosmos.slashing.v1beta1.Query/SigningInfo",
+
+	//staking
+	"/cosmos.staking.v1beta1.Query/Delegation",
+	"/cosmos.staking.v1beta1.Query/Params",
+	"/cosmos.staking.v1beta1.Query/Validator",
+
+	//osmosis queries
+
+	//epochs
+	"/osmosis.epochs.v1beta1.Query/EpochInfos",
+	"/osmosis.epochs.v1beta1.Query/CurrentEpoch",
+
+	//gamm
+	"/osmosis.gamm.v1beta1.Query/NumPools",
```

Comment:
Maybe we should not whitelist it as it is deprecated?

Reply:
Yeah, I think it makes sense to remove the deprecated queries. Is it just NumPools and EstimateSwapExactAmountIn/Out. Are there any replacements for those?

Reply:
Yes, the same queries but in the `x/poolmanager` module